src/resources/groovy/grape/defaultGrapeConfig.xml

```diff
@@ -26,7 +26,7 @@
         <ivy pattern="${user.home}/.groovy/grapes/[organisation]/[module]/ivy-[revision].xml"/>
         <artifact pattern="${user.home}/.groovy/grapes/[organisation]/[module]/[type]s/[artifact]-[revision](-[classifier]).[ext]"/>
       </filesystem>
-      <ibiblio name="localm2" root="file:${user.home}/.m2/repository/" checkmodified="true" changingPattern=".*" changingMatcher="regexp" m2compatible="true"/>
+      <ibiblio name="localm2" root="file:/${user.home}/.m2/repository/" checkmodified="true" changingPattern=".*" changingMatcher="regexp" m2compatible="true"/>
```

Comment:
Actually, as noted in the RFC[1] and the simplified version[2], this should be:

```
file:///${user.home}/.m2/repository/
```
That should work fine both on Windows and *nix

[1] https://tools.ietf.org/html/rfc8089
[2] https://en.wikipedia.org/wiki/File_URI_scheme

Reply:
I tried with file:/// and it is not working:

cachedGrapes: resource not reachable for org.apache.camel#camel-catalog-provider-karaf;2.19.2: res=C:\Users\Aurelien Pupier\.groovy\grapes\org.apache.camel\camel-catalog-provider-karaf\jars\camel-catalog-provider-karaf-2.19.2.jar
	 trying file:///C:\Users\Aurelien Pupier/.m2/repository/org/apache/camel/camel-catalog-provider-karaf/2.19.2/camel-catalog-provider-karaf-2.19.2.jar
		tried file:///C:\Users\Aurelien Pupier/.m2/repository/org/apache/camel/camel-catalog-provider-karaf/2.19.2/camel-catalog-provider-karaf-2.19.2.jar

Reply:
it is working with file://

Reply:
file:///${user.home}/.m2/repository/ 

is most likely not correct. You will end up with 4 slashes on Linux/Unix.

It should be file:// afaik

Reply:
besides that the change works on my fedora machine.

Reply:
just tested with file:// and it fails for me.

Reply:
to sum up:
- "file:/"
-- Windows OK
-- Linux OK
- "file://"
-- Windows OK
-- Linux KO
- "file:///"
-- Windows KO
-- Linux KO

and the winner is: "file:/"

Reply:
This seems logical/obvious since ${user.home} includes a prefix slash, and for a valid URI you need two slashes. Thus when in doubt, file:/${user.home}/path/to/things. This is true for p2 repo URIs too.